.github/workflows/pr-checks.yml

```diff
@@ -13,13 +13,17 @@ on:
     types:
       - milestoned
       - demilestoned
+
 concurrency:
   group: pr-checks-${{ github.event.number }}
+
+permissions:
+  checks: write
+  actions: write
+  contents: read
```

Comment:
Do you perhaps also need "pull-requests: read" here?

Reply:
probably, yeah